core/contractsdk/java/src/main/java/com/baidu/xuper/ContextImpl.java

```diff
@@ -99,6 +114,111 @@ public void deleteObject(byte[] key) {
         return;
     }
 
+//    @Override
+    public Contract.IteratorResponse newIterator(ByteString start, ByteString limit) {
+        Contract.IteratorRequest request = Contract.IteratorRequest.newBuilder().setHeader(this.header)
```

Comment:
Be careful using setXXX() with user input fields, nullable types could cause NullPointer exception.

Reply:
Sure, but it would throw an exception like "contract error status:500 message:java.lang.NullPointerException" to the contract's caller. 
(1)The exception could not disturb the operation of XuperChain.
(2)The caller could solve the exception by checking the contract.